slauger/check_netscaler
# Release 1.2.0

- removed whitespace from sub check_server
- added links to contributors in Markdown syntax (GitHub does not recognize the syntax with ‘@user')
- fixed typo (exmaples)
- release v1.2.0
- prepare for v1.2.0, harmonize string quoting
- added new commands to plugin_test.sh
- changed description for service group check
- refactored the documentation
- Merge pull request #19 from bb-Ricardo/master
- enhancing plugin, adding Icinga2 examples
- fixed typo in icinga2 templates file
- Added Icinga2 service examples
- added new commands to Icinga2 templates
- adding cli option -x (urlopts)
- Adding command line option -x / --urlopts to set NITRO URL options for further filtering in debug mode
- added ICA User session example to README
- make use of nagios_die
- I like the name of the function
- merged check_string and check_string_not into one function
- Updated README with latest changes
- updated CHANGELOG for servicegroup command
- added member quorum check to servicegroup command
- now you can define (in percent) how many servicegroup members can go DOWN until you get a WARNING or CRITICAL output
- enhanced performance data output for "perfromancedata"
- unified threshold functions to check_threshold
- added command servicegroup to README
- Update CHANGELOG.md
- Added new command servicegroup
- check the state of a servicegroup and its members
- fixed issue with interfaces perf data output
- performancedata command van now parse arrays
- Now it is possible to get performance data from arrays like Interface.
  - identifier and field name need to be separated by '.'
  - example Interface endpoint:
-  -C performancedata -o Interface -n id.tottxbytes,id.totrxbytes
- added a contributers section to README
- added "performancedata" command to README
- added command performancedata to Changelog
- added new option to gather performancedata
- added option "performancedata" to query stats and counters to forward them to a performance tool of your choice
- add IP adresses to server out in check_netscaler
- Update README.md
- Added new commands to README
- Changelog Update
- Add version information to main script
- added three new commands to check_netscaler.pl
- Added following commands:
  - server -> check status of Load Balancing Servers
  - hwinfo -> just print information about the Netscaler itself
  - interfaces -> check state of all interfaces and add performance data for each interface
- removing trailing white spaces
- fixed templates description
- added icinga2 service templates
- Create icinga2_command.conf
- Added Icinga2 command definition
